src/benchmarks/gc/GCPerfSim/GCPerfSim.cs

```diff
@@ -533,6 +541,26 @@ public void Free()
 
     public byte[] GetPayload() =>
         Util.NonNull(payload);
+
+    class ItemFinalizable : Item
```

Comment:
Can we make this class and the class below internal sealed?

Reply:
The whole file is using default internal accessibility.  For sealed, I should change the ReferenceItem analogue as well.  I'm ambivalent on this since it's a single file project and they don't necessarily need to be sealed, but also they don't need to be unsealed and this documents that there are no further subclasses, so sure.